src/recommendations/ProductCard/index.jsx

```diff
@@ -82,8 +83,10 @@ const ProductCard = ({
       productTypeCopy={productTypeCopy}
       productType={productType}
       variant={variant}
+      isLoading={isLoading}
       footer={(
         <Footer
+          isLoading={isLoading}
```

Comment:
Why do we need `isLoading` in `Footer`?

Reply:
oh that was for some random testing, i will remove it.